tests/spec/fs.copyFile.spec.js

```diff
@@ -0,0 +1,34 @@
+var util = require('../lib/test-utils.js');
+var expect = require('chai').expect;
+const fs = require('fs');
+const {COPYFILE_EXCL} = fs.constants;
+
+describe('fs.copyFile', function() {
```

Comment:
Let's add `.skip` to this, since we don't have an implementation yet.

https://mochajs.org/#inclusive-tests